.github/actions/build-docker/action.yml

```diff
@@ -0,0 +1,53 @@
+name: 'Build Quickwit docker image'
+description: 'Build & push docker image'
+inputs:
+  version:
+    description: 'Version'
+    required: false
+runs:
+  using: "composite"
+  steps:
+    - name: Image name when version is not empty
+      if: "${{ inputs.version != '' }}"
+      run: echo "IMAGE_ID=quickwit/quickwit-${{ inputs.version }}" >> $GITHUB_ENV
+      shell: bash
+    - name: Image name when version is empty
+      if: "${{ inputs.version == '' }}"
+      run: echo "IMAGE_ID=quickwit/quickwit" >> $GITHUB_ENV
+      shell: bash
+    - run: echo "IMAGE_DATE=$(date '+%Y-%m-%d')" >> $GITHUB_ENV
+      shell: bash
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
+        cache: 'yarn'
+        cache-dependency-path: quickwit-ui/yarn.lock
+    - run: make build-ui
```

Comment:
For this build, I guess it would be nice to include the UI build inside the Dockerfile as we are doing everything there.

Reply:
ok